packages/shared/src/logger.ts

```diff
@@ -11,6 +11,7 @@ export const isDebug = () => {
     return false;
```

Comment:
We have a condition judgement here, so it will only affect the performance when `process.env.DEBUG` exists